src/components/Row/Row.css

```diff
@@ -0,0 +1,45 @@
+.row{
+    margin-left: 10px;
+}
+
+.row_posters{
+    display: flex;
+    overflow-y: hidden;
+    overflow-x: scroll;
+    padding: 20px;
+    
+}
+
+.row_posters::-webkit-scrollbar{
+    display: none;
+}
+
+.row_poster{
+    object-fit: contain;
+    width: 100%;
+    max-height: 100px;
+    transition: transform 450ms;
+    margin-right: 20px;
+}
+
+.row_poster:hover{
+    transform: scale(1.15);
+
+}
+.row_title{
+    color: white;
+    text-align: left;
+    margin-top: 0px;
+    margin-bottom: 0px;
```

Comment:
you can remove margin-top:0 if not necessary

Reply:
Done in Upcoming PR